src/Form/EventType.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class EventType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name', TextType::class, ['required' => true])
+            ->add('description', TextareaType::class, ['required' => true])
+            ->add('address', TextType::class, ['required' => true])
+            ->add('startAt', DateType::class, ['required' => true])
```

Comment:
I can't set a date in past, so maybe remove all past days/month/years in past in choices

Same for end date

Reply:
I don't know if that's possible with the DateType form, I will check

Reply:
You should use html5 datepicker like in https://github.com/DarkmiraTour/community-event-manager/pull/158/files

Then you can set min and max date as html attributes by doing `'attr' => ['min' => new \DateTime(...), 'max' => ...]`